inc/saiport.h

```diff
@@ -2520,6 +2574,23 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_POE_PORT_ID,
 
+    /**
+     * @brief Port error event
+     *
+     * @type sai_port_error_event_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_ERROR_EVENT,
+
+    /**
+     * @brief Enable/Disable Port UNRELIABLE loss of signal
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_UNRELIABLE_LOS_ENABLE,
+
```

Comment:
@JaiOCP This Attribute does not seem to aligned with the PR description for error events. This is more of capability of the PHY which can support unreliable LOS enable. So, can we take this in a separate PR?